src/main/java/org/terasology/utilities/procedural/RegionSelectorNoise.java

```diff
@@ -61,7 +61,7 @@ public RegionSelectorNoise(long seed, int gridSize, float xCenter, float yCenter
     public float noise(int x, int y) {
         Vector2f relative = new Vector2f((float) x - xCenter, (float) y - yCenter);
 
-        if (relative.equals(Vector2f.zero())) {
+        if (relative.equals(new Vector2f(0, 0))) {

```

Comment:
this will always be invalid.

Reply:
Really? Even in the case where `x == xCenter`  and `y == yCenter`, i.e., we do a lookup at the center position of this region?

Reply:
Floating point precision will ensure they are never equal.

Reply:
:thinking: I see how this might be a problem due to inaccuracies, but I would have assumed that this would still workout in most cases. 
How would you improve/fix this part of the code?